state/execution_test.go

```diff
@@ -136,9 +136,9 @@ func TestBeginBlockByzantineValidators(t *testing.T) {
 			types.TM2PB.Evidence(ev2, valSet, now)}},
 	}
 
-	vote0 := &types.Vote{ValidatorIndex: 0, Timestamp: now, Type: types.PrecommitType}
-	vote1 := &types.Vote{ValidatorIndex: 1, Timestamp: now}
-	votes := []*types.Vote{vote0, vote1}
+	vote0 := (&types.Vote{ValidatorIndex: 0, Timestamp: now, Type: types.PrecommitType}).CommitSig()
+	vote1 := (&types.Vote{ValidatorIndex: 1, Timestamp: now}).CommitSig()
+	votes := []*types.CommitSig{vote0, vote1}
```

Comment:
nit: could also be renamed to `commit0`, `commit1`, `commit2` (or precommit0 etc)